docs/user-manual.md

```diff
@@ -115,47 +114,50 @@ To reboot a node without removing volumes, follow these steps:
 3. Run `kubectl uncordon NODE` after the node comes back online.
 4. After reboot, Pods will be rescheduled to the same node because PVCs remain intact.
 
-Inline Ephemeral Volumes (**deprecated**)
-------------------------
+### Generic Ephemeral Volume
 
-An example use of a TopoLVM inline ephemeral volume is as follows:
+TopoLVM supports the Generic Ephemeral Volume feature.  
+You can use Generic Ephemeral Volumes with TopoLVM like following:
 
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
-  name: ubuntu
+  name: my-pod-ephemeral
+  namespace: default
   labels:
-    app.kubernetes.io/name: ubuntu
+    app.kubernetes.io/name: my-pod-ephemeral
+    app: example
 spec:
   containers:
   - name: ubuntu
-    image: nginx4
+    image: quay.io/cybozu/ubuntu:20.04
+    command: ["/usr/local/bin/pause"]
     volumeMounts:
     - mountPath: /test1
       name: my-volume
   volumes:
   - name: my-volume
-    csi:
-      driver: topolvm.cybozu.com
-      fsType: xfs
-      volumeAttributes:
-          topolvm.cybozu.com/size: "2"
+    ephemeral:
+      volumeClaimTemplate:
+        spec:
+          accessModes:
+          - ReadWriteOnce
+          resources:
+            requests:
+              storage: 1Gi
+          storageClassName: topolvm-provisioner
 ```
 
-`driver` must be `topolvm.cybozu.com`.
+By using the Generic Ephemeral Volume function, you can use any CSI driver in the container to utilize the temporary volume.  
+TopoLVM schedules also based on the capacity used by the Generic Ephemeral Volumes.  
+When using Generic Ephemeral Volumes, the following processing is performed:
 
-`fsType` is optional.  To specify a filesystem type, give
-`csi.storage.k8s.io/fstype` parameter. If no type is specified, the default
-of ext4 will be used.
-`volumeAttributes` are optional.  To specify a volume size, give
-`topolvm.cybozu.com/size` parameter. If no size is specified, the default of
-1 GiB will be used.
+- When applying a Pod with a Generic Ephemeral Volume, the ephemeralController in kube-controller-manager creates a PVC for the Generic Ephemeral Volume(that is, it works the same as creating a regular PVC).
+- When the pod with a Generic Ephemeral Volume is deleted, the PVC is also deleted at the same time because if PVC is created as a Generic Ephemeral Volume, the PVC's OwnerReference is set to the Pod associated.
+- Since the deletion of PVs and real volumes associated with PVCs depends on ReclaimPolicy setting of StorageClass, StorageClass used in Generic Ephemeral Volume must be set to `reclaimPolicy=Delete` if you want to delete PVs and real volumes associated when delete the Pod.
```

Comment:
```suggestion
- Since the deletion of PVs and real volumes associated with PVCs depends on ReclaimPolicy setting of StorageClass, StorageClass used in Generic Ephemeral Volume must be set to `reclaimPolicy=Delete` if you want to delete PVs and real volumes associated when deleting the Pod.
```